packages/client/src/navigation/routes.ts

```diff
@@ -43,18 +43,18 @@ export const VERIFY_CORRECTOR = '/correction/:declarationId/verify/:corrector'
 export const SEARCH = '/search'
 export const SEARCH_RESULT = '/search-result'
 export const CERTIFICATE_COLLECTOR =
-  '/cert/collector/:registrationId/:eventType/:groupId'
```

Comment:
Do we have to have the template ID in the URL? It's stored in the declaration details too (that get sent to the backend), right?